docs/workflows/alternate-content-source.rst

```diff
@@ -0,0 +1,46 @@
+Alternate Content Sources
+=========================
+
+To use an Alternate Content Source you need a ``RPMRemote`` with path of your ACS.
+
+.. warning::
+
+    Mirror list cannot be used as an Alternative Content Source.
+
+.. code-block:: bash
+
+    http POST localhost:24817/pulp/api/v3/remotes/rpm/rpm/ name="myACS" policy="on_demand" url="http://fixtures.pulpproject.org/rpm-unsigned/"
```

Comment:
Other documentation uses `$BASE_ADDR`, we should probably keep it consistent: https://github.com/pulp/pulp_rpm/blob/master/docs/workflows/create_sync_publish.rst